### ISSUE TITLE
Should we add a news file?

### ISSUE THREAD

**Issue by kakra** [Zygo/bees#50]
Since there seems to be interesting progress with each release, should we add a NEWS file to the project root? Maybe listing contributors and important issues fixed...

**Comment by Zygo** [comment]
Good idea.  We should start by compiling the annotated tag commit messages...

**Comment by kakra** [comment]
Okay, I queued up an initial idea in https://github.com/kakra/bees/tree/integration.

Please don't pull from that branch, it will be force-pushed on many occassions. But feel free to cherry-pick.

**Comment by Zygo** [comment]
OK, I'll pick instead of merging /integration.

I picked several obvious improvements from /integration as well as two of your branches.  I'll look at the rest in more depth later (they are too non-trivial for a Saturday afternoon ;).
